public/js/solution_submit.js

```diff
@@ -60,3 +60,7 @@ const submitSolution = () => {
 		});
 	});
 }
+
+const goBack = () => {
+	window.location.href = "/problem/" + question_id ;
+}
```

Comment:
Add newline.

Reply:
> This'll be a better way @vsvipul ,what do you say? `<button type="button" onclick="javascript:history.back()">Cancel</button>`

but what if the user directly goes to the submit page without any history? isn't it better to code hard links ?